### PR TITLE
Add Huber loss

### LINKED ISSUE #871: HuberLoss support

### PR DESCRIPTION
**Pull request by pglusacTT**
Add support for Huber loss

Closes #871

### CODE REVIEW

File: forge/forge/op/loss.py [hunk 1]
```diff
@@ -119,3 +120,47 @@ def forward(self, predictions, labels):
         square = Multiply("square", diff, diff)
         loss = reduce_loss(self.reduction, square)
         return loss
+
+
+class HuberLoss(ForgeModule):
+    def __init__(self, name: str, delta: float = 1.0, reduction: str = "avg"):
```
Comment (pmarkovicTT) [inline forge/forge/op/loss.py:126]: Should default value for reduction be "mean"?

File: forge/forge/op/loss.py [hunk 2]
```diff
@@ -119,3 +120,51 @@ def forward(self, predictions, labels):
         square = Multiply("square", diff, diff)
         loss = reduce_loss(self.reduction, square)
         return loss
+
+
+class HuberLoss(ForgeModule):
+    def __init__(self, name: str, delta: float = 1.0, reduction: str = "mean"):
```
Comment (pmarkovicTT) [inline forge/forge/op/loss.py:126]: nit: add description here as well

**Review by pmarkovicTT — APPROVED** [review]
lgtm

**Review by pilkicTT — APPROVED** [review]
